client/blocks/simple-site-rename-form/index.jsx

```diff
@@ -39,14 +44,44 @@ export class SimpleSiteRenameForm extends Component {
 		this.props.requestSiteRename( selectedSiteId, this.state.domainFieldValue, discard );
 	};
 
+	getDomainValidationMessage( domain ) {
+		const { translate } = this.props;
+
+		if ( isEmpty( domain ) ) {
+			return '';
+		}
+
+		if ( domain.match( /[^a-z0-9]/i ) ) {
+			return translate( 'Domain can only contain letters (a-z) and numbers.' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **85** times:
`translate( 'Usernames can only contain lowercase letters (a-z) and numbers.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).